### ISSUE TITLE
SyntaxError: invalid token

### ISSUE THREAD

**Issue by ser** [costela/wile#2]
```
# pip3 install wile
# wile
Traceback (most recent call last):
  File "/usr/local/bin/wile", line 7, in <module>
    from wile import main
  File "/usr/local/lib/python3.5/dist-packages/wile.py", line 70
    os.makedirs(os.path.dirname(account_key_path), 0750)
                                                      ^
SyntaxError: invalid token
```

**Comment by costela** [comment]
Unfortunately wile is python2 only for now, so only `pip install wile` should work. I'm working on making it version-agnostic though.

To be honest I don't even remember why I wrote it using python2 ... there must have been a reason :)

**Comment by ser** [comment]
```
# wile
Traceback (most recent call last):
  File "/usr/local/bin/wile", line 7, in <module>
    from wile import main
  File "/usr/local/lib/python2.7/dist-packages/wile.py", line 6, in <module>
    from setuptools_scm import get_version
ImportError: No module named setuptools_scm
```

**Comment by costela** [comment]
oh, that's a different issue. Should be fixed in the just uploaded 0.1.4. Thanks for the catch!

**Comment by ser** [comment]
Thanks, it works now. At least at first run :) I will report any issues if I meet them :)

**Comment by ser** [comment]
Yup, it works, I have my certs :)

The only thing I would suggest is to create some meaningful error informations, as currently wile just throws traceback. For example I got `UnboundLocalError: local variable 'crt' referenced before assignment` when my web server was unreachable from letsencrypt side (by my mistake, of course)

**Comment by costela** [comment]
Coincidently I was just working on this problem. Also fixed with 5364f248. I'm gonna try to work on #1 before releasing 0.1.5 though.
